templates/openstack/resources.html

```diff
@@ -0,0 +1,345 @@
+    </p>
+  </div>
+</section>
+
+<div class="u-fixed-width">
+  <hr class="p-separator">
+</div>
+
+<section class="p-strip u-no-padding--top u-no-padding--bottom">
+  <div class="u-fixed-width">
+    <h2>OpenStack blogs and related topics</h2>
+    <p>This rich set of content covers everything from high-level product announcements to hands-on technical guides that dig into more complicated topics.</p>
+    <p>Scroll through our entire library of blogs about <a href="/blog/tag/openstack">OpenStack</a> and <a href="/blog/tag/private-cloud">private-cloud</a>.</p>
+  </div>
+</section>
+{% with section_classes='p-strip u-no-padding--bottom', heading_topic='OpenStack', tag_name='OpenStack', tag_id='1672', limit='4' %}
+  {% include "shared/_latest_news_strip.html" %}
+{% endwith %}
+
+<div class="u-fixed-width">
+  <hr class="p-separator">
+</div>
+
+<section class="p-strip u-no-padding--top">
+  <div class="u-fixed-width u-sv3">
+    <h2>More OpenStack resources</h2>
+    <p>Once you get started with OpenStack, start reading our library of resources, that includes tutorials and documentation on various topics.</p>
+  </div>
+  <div class="row p-divider">
+    <div class="col-6 p-divider__block">
+      <h3>Tutorials</h3>
+      <p>Starting with just your workstation, learn how to use OpenStack for cloud infrastructure implementation purposes, from a single-node installation to large-scale clusters using a series of tutorials.</p>
+      <p>
+        <a href="/openstack/tutorials">Go to the tutorials&nbsp;&rsaquo;</a>
+      </p>
+    </div>
+    <div class="col-6 p-divider__block">
+      <h3>Documentation</h3>
+      <p>Deploy your own infrastructure based on Charmed OpenStack and benefit from full automation and rich lifecycle management capabilities. Charmed OpenStack is a cost-effective cloud platform designed to host mission critical applications.</p>
+      <p>
+        <a href="/openstack/docs">Go to the documentation&nbsp;&rsaquo;</a>
+      </p>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip--light">
+  <div class="row">
+    <div class="col-7">
+      <h2>OpenStack pricing calculator</h2>
+      <h4>Should your workloads run in a public or private cloud?</h4>
+      <p>Depending on your requirements, overall CapEx and OpEx costs and cost-per-resource metrics, it may be more economical to run your workloads in the private cloud. Use the following private cloud pricing calculator to estimate the costs of running your instances in the Charmed OpenStack private cloud. And see how much you can save compared to the public clouds.</p>
+      <h3 class="p-heading--4">Number of instances
+        <span class="p-tooltip--top-left" aria-describedby="tp-lft">
+          <i class="tco-tooltip p-icon--help"></i>
+          <span class="p-tooltip__message" role="tooltip" id="default-tooltip">A number of virtual machines that will run on the cloud.</span>
+        </span>
```

Comment:
These values should match, this is what allows screenreaders to pick up the tooltip value:

```suggestion
        <span class="p-tooltip--top-left" aria-describedby="vm-tooltip">
          <i class="tco-tooltip p-icon--help"></i>
          <span class="p-tooltip__message" role="tooltip" id="vm-tooltip">A number of virtual machines that will run on the cloud.</span>
        </span>
```